spec/integration_spec.rb

```diff
@@ -83,6 +90,33 @@ def app
     end
   end
 
+  describe('get /guides/:gid/exercises/:eid/test') do
+    describe 'run tests for specific guide\'s exercise' do
+      describe 'When guide exists' do
```

Comment:
I would use `context` instead of `describe`, since you are describing an scenario instead of a feature. 

The rumb of thum is: _if you message starts with **when**  , then use `context`_